tests/testthat/test_cloneVariable.R

```diff
@@ -19,6 +19,16 @@ test_that("Clone variable", {
 
 })
 
+test_that("Clone variable with invalid variable name", {
+  out <- cloneVariable(dfSAV, varName = "VAR1", new_varName = "Alter", checkVarName = FALSE)
+  expect_equal(namesGADS(out), c("VAR1", "VAR2", "VAR3", "Alter"))
+  expect_equal(out$dat$VAR1, out$dat$Alter)
```

Comment:
In a comment for the code or in the message raised by the function?

Reply:
Maybe in the function.... But this would probably be a larger fix (and might not be important enough as it might only affect Alter and ALTER as a special case).

Reply:
Sounds very good to me!